frontend/src/components/cluster/Overview.tsx

```diff
@@ -136,12 +165,14 @@ function EventsSection() {
               >
                 <Box>{makeStatusLabel(event)}</Box>
               </LightTooltip>,
+            sort: false
           },
         ]
           :
           []
         }
         data={events ? events.sort(sortFunc) : null}
+        isDataAlreadySortedInOneOfTheOrder
```

Comment:
Instead of this property, maybe a more flexible and simpler way to do this is to just set whether we need a table to be sorted by a table. For example: `defaultSortingColumn: number`. This way, if it's set to `defaultSortingColumn: -2`, we can sort decendingly (or ascendingly if it's `2`) using the 2nd column (we cannot use 0 because we cannot use minus with it).